tests/jax/test_scalar_types.py

```diff
@@ -0,0 +1,17 @@
+# SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import jax
+import pytest
+from infra import run_op_test_with_random_inputs
+
+# TODO not sure what this is supposed to test. Rethink.
```

Comment:
Change to `TODO We should put here tests which operate on scalars, with different types of ops (for example one unary, one binary etc). Test below is remnant from the old test set, reconsider keeping it.`